frontend/appflowy_flutter/lib/plugins/document/application/document_data_pb_extension.dart

```diff
@@ -62,8 +62,8 @@ extension DocumentDataPBFromTo on DocumentDataPB {
     blocks.forEach((key, value) {
       final parentId = value.parentId;
       if (parentId.isNotEmpty) {
-        childrenMap[parentId] ??= ChildrenPB.create();
-        childrenMap[parentId]!.children.add(value.id);
+        childrenMap[blocks[parentId]!.childrenId] ??= ChildrenPB.create();
```

Comment:
Don't force unwrap the values. It may lead to unexpected runtime errors.

Reply:
I have added a check in the if condition to check is blocks[parentId] is not null